index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+
+    <meta name="description" content="판다마켓 - Sprint-Mission 1" />
+    <meta name="keywords" content="판다마켓" />
+    <meta name="author" content="Codeit Sprint FrontEnd" />
+
+    <meta property="og:title" content="판다마켓" />
+    <meta property="og:description" content="판다마켓 - Sprint-Mission 1" />
+    <!-- <meta property="og:image" content="배포 후 추가 예정" /> -->
+    <!-- <meta property="og:url" content="배포 후 추가 예정"/> -->
+
+    <link rel="icon" href="./assets/favicon.ico" type="image/x-icon" />
+
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable-dynamic-subset.min.css" />
+
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/typography.css" />
+    <link rel="stylesheet" href="./styles/header.css" />
+    <link rel="stylesheet" href="./styles/banner.css" />
+    <link rel="stylesheet" href="./styles/info.css" />
+    <link rel="stylesheet" href="./styles/footer.css" />
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <header class="header">
+      <a href="/" class="header__logo">
+        <picture>
+          <source media="(max-width: 480px)" srcset="./assets/logos/logo_typo.png" />
+          <img src="./assets/logos/logo_sm.png" alt="판다마켓 로고" />
+        </picture>
+      </a>
+      <a href="/login" class="header__login font-semibold text-lg">로그인</a>
```

Comment:
❗️ 수정요청
현재 코드에서 태그를 클릭하면 notFound 에러가 발생하고 있습니다.
이는 "/login" 경로에 해당하는 파일을 찾을 수 없기 때문인데요.
이를 해결하는 방법은 여러 가지가 있지만, 지금 단계에서는 확장자를 명시적으로 추가하는 방법을 추천드립니다.

지섭님이 코딩에 능숙하신 것 같아서 추천드리자면 왜 이렇게 동작하는지 고민해보세요~
그리고 작업을 하신 후 클릭 가능한 요소들은 직접 만져보시면서 테스트 하시는 것을 추천드립니다~

```suggestion
      <a href="/login.html" class="header__login font-semibold text-lg">로그인</a>
```

Reply:
빈페이지로 이동하라는 것이 빈 html로 이동하라는 것인지 모르고 라우트로만 해결하려고 했었습니다..!
미션2때 반영해두도록 하겠습니다!